lumicks/pylake/kymotracker/tests/test_derived_quantities/test_msd.py

```diff
@@ -440,12 +444,14 @@ def test_cve_skipped_samples(
 
 
 @pytest.mark.parametrize(
-    "diffusion,obs_noise,num_points,time_step,blur_constant,"
-    "diffusion_ref,var_diffusion_ref,num_points_ref,loc_var_ref",
+    "diffusion,obs_noise,num_points,time_step,blur_constant,loc_var,loc_var_var,"
+    "diffusion_ref,var_diffusion_ref,num_points_ref,loc_var_ref, loc_var_var_ref",
```

Comment:
Naming (see above). `_ref` is good, though!

Reply:
👍